spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/RowDataReader.java

```diff
@@ -39,22 +36,22 @@
 import org.apache.iceberg.parquet.Parquet;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.collect.Sets;
-import org.apache.iceberg.spark.SparkSchemaUtil;
 import org.apache.iceberg.spark.data.SparkAvroReader;
 import org.apache.iceberg.spark.data.SparkOrcReader;
 import org.apache.iceberg.spark.data.SparkParquetReaders;
 import org.apache.iceberg.types.TypeUtil;
 import org.apache.spark.rdd.InputFileBlockHolder;
 import org.apache.spark.sql.catalyst.InternalRow;
 
-class RowDataReader extends BaseDataReader<InternalRow> {
+class RowDataReader<CST extends ContentScanTask<?>, G extends ScanTaskGroup<CST>>
```

Comment:
Is RowDataReader only for FileScanTask?  We should just provide the parameter to BaseDataReader right?

Reply:
No. RowDataReader will also be inherited by cdc readers.